src/AutoRest.CSharp.V3/Properties/launchSettings.json

```diff
@@ -1,12 +1,12 @@
 {
   "profiles": {
     "additionalProperties": {
-      "commandLineArgs": "--standalone --input-codemodel=$(SolutionDir)\\test\\TestServerProjects\\additionalProperties\\CodeModel.yaml --plugin=csharpgen --output-folder=$(SolutionDir)\\test\\TestServerProjects\\additionalProperties --namespace=additionalProperties --shared-source-folder=$(SolutionDir)\\src\\assets --save-code-model=true",
-      "commandName": "Project"
+      "commandName": "Project",
```

Comment:
Argh, why did it reorder again?

Reply:
I think you're looking at an old comparison. The only thing that is added (after my commit) was the media_types into the correct place.